### PR TITLE
Bug Fix

### PR DESCRIPTION
**Pull request by Thomas-Gallant**
Changelog:
- `start.sh` now uses a more accurate method of counting lines in `puzzles.sql` and `puzzles.txt`
    - https://stackoverflow.com/questions/16533748/the-wc-l-gives-wrong-result

### CODE REVIEW

File: delete.sh
Comment (Gregory711) [inline delete.sh:1]: I don't think we want to do this, the point of delete.sh was to clear the database so a rerun would be from a fresh start e.g. if user exec into the db and ran sql queries. puzzles.sql is part of the git repo, if user wants to reset that they could just do git checkout -- puzzles.sql

File: puzzles.sql
Comment (Gregory711) [inline puzzles.sql:1]: This data shouldn't be removed, it is the already solved puzzles stored so we can have puzzles in the db at startup and not have to resolve all puzzles everytime.

File: puzzles.txt
Comment (Gregory711) [inline puzzles.txt:1]: Adding more puzzles is good but we should keep puzzles.sql in sync so startup time is still fast.

File: delete.sh
Comment (Thomas-Gallant) [reply delete.sh:1]: I've realized taking another look at it my proposed changes don't make sense, thanks for catching that!

**Review by Gregory711 — APPROVED** [review]

**Pull request merged**